src/moonbeam/moonbase-local.js

```diff
@@ -1,287 +1,32 @@
-const CONTRACT_INPUT = '0xd09de08a';
-
-const keyring = new Keyring({ type: 'sr25519' });
-
-// Alith is the default sodo wallet on Moonbase Local
-const MoonbaseAlith = {
-    name: 'Alith',
-    privateKey: '0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133',
-};
-
-// Alice is the default sodo wallet on Turing Dev
-const TuringAlice = {
-    name: 'Alice',
-    privateKey: '0xe5be9a5092b81bca64be81d212e7f2f9eba183bb7a90954f7b76361f6edb5c0a',
-};
-
-const createEthereumXcmTransactThroughProxyExtrinsic = (parachainHelper, transactAs) => {
-    const extrinsic = parachainHelper.api.tx.ethereumXcm.transactThroughProxy(
-        transactAs,
-        {
-            V2: {
-                gasLimit: 71000,
-                action: { Call: CONTRACT_ADDRESS },
-                value: 0,
-                input: CONTRACT_INPUT,
-            },
-        },
-    );
-    return extrinsic;
-};
-
-const createAutomationTaskExtrinsic = ({
-    turingHelper, schedule, parachainId, scheduleFee, payloadExtrinsic, payloadExtrinsicWeight, overallWeight, fee, scheduleAs,
-}) => {
-    const extrinsic = turingHelper.api.tx.automationTime.scheduleXcmpTaskThroughProxy(
-        schedule,
-        { V3: { parents: 1, interior: { X1: { Parachain: parachainId } } } },
-        scheduleFee,
-        { asset_location: { V3: { parents: 1, interior: { X2: [{ Parachain: parachainId }, { PalletInstance: 3 }] } } }, amount: fee },
-        payloadExtrinsic.method.toHex(),
-        payloadExtrinsicWeight,
-        overallWeight,
-        scheduleAs,
-    );
-
-    console.log(`Task extrinsic encoded call data: ${extrinsic.method.toHex()}`);
-    return extrinsic;
-};
-
-const scheduleTaskFromMoonbase = async ({
-    turingHelper, parachainHelper, taskExtrinsic, taskExtrinsicPaymentInfo, keyPair,
-}) => {
-    const encodedTaskExtrinsic = taskExtrinsic.method.toHex();
-    const transactCallWeight = taskExtrinsicPaymentInfo.weight;
-
-    console.log(`Proxy task extrinsic encoded call data: ${encodedTaskExtrinsic}`);
-    console.log('transactCallWeight: ', transactCallWeight.toHuman());
-
-    const overallWeight = turingHelper.calculateXcmTransactOverallWeight(transactCallWeight);
-    const fee = await turingHelper.weightToFee(overallWeight, parachainHelper.getNativeAssetLocation());
-
-    console.log(`\nExecute the above an XCM from ${parachainHelper.config.name} to schedule a task on ${turingHelper.config.name} ...`);
-
-    const transactExtrinsic = await parachainHelper.createTransactExtrinsic({
-        targetParaId: turingHelper.config.paraId,
-        encodedCall: encodedTaskExtrinsic,
-        callWeight: transactCallWeight,
-        overallWeight,
-        fee,
-    });
-
-    console.log(`transactExtrinsic Encoded call data: ${transactExtrinsic.method.toHex()}`);
-
-    await sendExtrinsic(parachainHelper.api, transactExtrinsic, keyPair);
-};
+const ALITH_PRIVATE_KEY = '0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133';
```

Comment:
Add **a code comment above this line** to show the original github link where this private key is already exposed.

Reply:
Fixed